storage/client/client.go

```diff
@@ -1202,26 +1202,18 @@ func (c *StorageClient) RuntimeTokens(ctx context.Context, p apiTypes.GetRuntime
 	}
 	for res.rows.Next() {
 		var t EvmToken
-		var totalSupplyNum pgtype.Numeric
 		if err2 := res.rows.Scan(
 			&t.ContractAddr,
 			&t.EvmContractAddr,
 			&t.Name,
 			&t.Symbol,
 			&t.Decimals,
-			&totalSupplyNum,
+			&t.TotalSupply,
 			&t.Type,
 			&t.NumHolders,
 		); err2 != nil {
 			return nil, wrapError(err)
 		}
-		if totalSupplyNum.Status == pgtype.Present {
```

Comment:
v4 had a bug where it didn't support scanning into _pointers to_ custom types. v5 does that just fine, so we simplify here.